src/convertToObject.js

```diff
@@ -6,7 +6,40 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resObj = {};
+  const cleanedSourceString = sourceString.replace(/\t+/, ' ').trim();
+  const styleStrings = cleanedSourceString.split(';');
+  let currentProperty = '';
+  let currentValue = '';
+
+  styleStrings.forEach((styleString) => {
+    if (styleString.trim() === '') {
+      return;
+    }
+
+    if (styleString.includes(':')) {
+      const index = styleString.indexOf(':');
+      const property = styleString.slice(0, index).trim();
+      const value = styleString.slice(index + 1).trim();
+
+      if (currentProperty) {
+        resObj[currentProperty] = currentValue.trim();
+        currentProperty = '';
+        currentValue = '';
+      }
+
+      currentProperty = property;
+      currentValue = value;
+    } else if (currentProperty) {
+      currentValue += ' ' + styleString.trim();
```

Comment:
Prefer template literals over concatenation https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
And you actually don't need these variables `currentProperty/currentValue`